feat(loading): match Loading component style

GuichiZhao opened the pull request:
update the existing Loading component, with conrrespondding doc and test

## Checklist

- [ ] Fix linting errors
- [ ] Tests have been added / updated (or snapshots)

## Change information

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/conflux-chain/react-ui/33)
<!-- Reviewable:end -->

GuichiZhao committed:
feat(loading): match Loading component style

update the existing Loading component, with conrrespondding doc and test

AngeliaQ commented on components/loading/loading.tsx:
```diff
@@ -51,7 +55,7 @@ const Loading: React.FC<React.PropsWithChildren<LoadingProps>> = ({
   const theme = useTheme()
   const width = useMemo(() => getIconSize(size), [size])
-  const bgColor = useMemo(() => getIconBgColor(type, theme.palette, color), [
+  color = useMemo(() => getIconStrokeColor(type, theme.palette, color), [
```
maybe it is better to use a new variable ·const strokeColor·

GuichiZhao committed:
docs(loading): add missing supported types

AngeliaQ commented on components/loading/loading.tsx:
```diff
@@ -73,16 +85,6 @@ const Loading: React.FC<React.PropsWithChildren<LoadingProps>> = ({
           height: 100%;
         }
-
-        label {
```
it should not delete

AngeliaQ commented on components/loading/loading.tsx:
```diff
@@ -73,16 +85,6 @@ const Loading: React.FC<React.PropsWithChildren<LoadingProps>> = ({
-          color: ${theme.palette.accents_5};
-        }
-
-        label :global(*) {
```
the same as above

GuichiZhao committed:
feat(loading): overload color and size props

Type checking still come into play internally although there is no limit for the client

GuichiZhao committed:
fix(loading): fix aAccent color to cNeutual
